app/uk/gov/hmrc/exports/models/PointerMapping.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.models
+
+case class PointerMapping(wcoPattern: PointerPattern, exportsPattern: PointerPattern) {
+
+  def applyToWCOPointer(pointer: Pointer): Pointer = {
+    if (!wcoPattern.matches(pointer.pattern))
```

Comment:
How are we thinking of obtaining all the wco patterns?

Reply:
Steve G has a mapping file, perhaps we can use the work you did in the wco-dec eventually to at least generate the left hand side of the mapping (`42A.26B.123.ABC` bit)
That way we'd know if we need to be handling it based on the schema